terra-jupyter-bioconductor/Dockerfile

```diff
@@ -7,4 +7,6 @@ RUN curl -O https://raw.githubusercontent.com/Bioconductor/anvil-docker/master/a
 	&& R -f install.R \
 	&& rm -rf install.R
 
+RUN chown -R $USER:users /opt/conda
```

Comment:
does this need to be in each individual image even tho it's already in base image?

Reply:
I'm uncertain, but I think it's safer this way. There are additional packages that get installed as the root user throughout the Dockerfiles.

Reply:
are you saying `pip` install will install packages to `/opt/conda`?

Reply:
Yes, if you run `conda list` It will show the source as pip instead of conda-forge

Reply:
can you cd into `/opt/conda` to confirm? just curious if the packages are indeed in this dir, or it's just `conda` is looking at `pip` package location as well

Reply:
Pip is also installed under /opt/conda/bin/pip, as well as the packages being located under /opt/conda/lib/python3.7/site-packages/

Requests is under that directory, and that is installed via pip3 in the dockerfile.

Reply:
I can test if it's necessary to run the chown in the latter Dockerfiles if you would like. I'm uncertain if it's necessary, it'll just take me a while to test.